phalcon/acl/adapterinterface.zep

```diff
@@ -38,6 +38,16 @@ interface AdapterInterface
 	public function getDefaultAction() -> int;
 
 	/**
+	 * Sets the default access level (Phalcon\Acl::ALLOW or Phalcon\Acl::DENY) for no arguments provided in isAllowed action if there exists func for accessKey
+	 */
+	public function setNoArgumentsDefaultAction(int defaultAccess);
+
+	/**
+	 * Returns the default ACL access level for no arguments provided in isAllowed action if there exists func for accessKey
+	 */
+	public function getNoArgumentsDefaultAction() -> int;
```

Comment:
You mean binary compability ? Cuz i dont know how it breaks backward compability. Well i can just do one commit for 2.1.x i guess.